packages/ns-threat_shield/files/20_threat_shield

```diff
@@ -1,13 +1,17 @@
 #!/bin/sh
 
 crontab -l | grep -q '/etc/init.d/ts-ip' || echo '50 * * * * sleep $(( RANDOM % 3500 )); /etc/init.d/ts-ip reload' >> /etc/crontabs/root
-crontab -l | grep -q '/etc/init.d/ts-dns' || echo '1 */12 * * * sleep $(( RANDOM % 3600 )); /etc/init.d/ts-dns reload' >> /etc/crontabs/root
+crontab -l | grep -q '/etc/init.d/ts-dns' || echo '1 */12 * * * sleep $(( RANDOM % 3600 )); /etc/init.d/adblock reload' >> /etc/crontabs/root
```

Comment:
```suggestion
crontab -l | grep -q '/etc/init.d/adblock' || echo '1 */12 * * * sleep $(( RANDOM % 3600 )); /etc/init.d/adblock reload' >> /etc/crontabs/root
```

Reply:
I was wondering if built-in `timer` action was better, but our cronjob is much cleaner.